tests/unit/lib/charms/kubernetes_charm_libraries/v0/test_multus.py

```diff
@@ -692,6 +713,23 @@ def test_given_no_nad_to_create_and_no_existing_nad_when_config_changed_then_cre
 
         patch_create_nad.assert_not_called()
 
+    @patch("lightkube.core.client.GenericSyncClient", new=Mock)
+    @patch(f"{MULTUS_LIBRARY_PATH}.KubernetesClient.list_network_attachment_definitions")
+    @patch(f"{MULTUS_LIBRARY_PATH}.KubernetesClient.patch_statefulset", new=Mock)
+    @patch(f"{MULTUS_LIBRARY_PATH}.KubernetesClient.statefulset_is_patched", new=Mock)
+    @patch(f"{MULTUS_LIBRARY_PATH}.KubernetesClient.create_network_attachment_definition")
+    @patch(f"{MULTUS_LIBRARY_PATH}.KubernetesClient.delete_network_attachment_definition")
+    def test_given_invalid_nads_to_create_when_config_changed_then_does_nothing(  # noqa: E501
```

Comment:
Do we want it to do nothing? I expected an error in this case.

Reply:
Raising an exception in the library side for invalid config is not desired. This is the workaround till the solution is implemented for https://warthogs.atlassian.net/browse/TELCO-759.

Reply:
Yes, sorry, you're right. Blocked status is what we want, and the charm is the right place to do this.